### PR TITLE
Make 'various' as default collection.

### LINKED ISSUE #16: Default collection is undefined.
When adding new item to db on admin page UI shows collection as already selected. But item gets added to an undefined collection. If user selects one of listed collections manually or adds new collection then all is fine.

### PR DESCRIPTION
**Pull request by g1st**
This should fix #16

### CODE REVIEW

File: components/Admin/AdminForm.js [hunk 1]
```diff
@@ -142,7 +142,8 @@ class AdminForm extends Component {
 
   // only when creating new item
   resetForm = () => {
-    const { collections } = this.props;
+    document.getElementById('images').value = null;
```
Comment (nikrb) [inline components/Admin/AdminForm.js]: this won't play nice with react? Can't set this be set in render function using props or state?

File: components/Admin/AdminForm.js [hunk 2]
```diff
@@ -152,14 +153,16 @@ class AdminForm extends Component {
...
-      existingCollection: collections[0] || '',
+      existingCollection: '',
       available: 'available'
     });
   };
 
   handleSubmit = e => {
     e.preventDefault();
-    this.setState(() => ({ updating: true }));
+    this.setState(() => ({
```
Comment (nikrb) [inline components/Admin/AdminForm.js]: original line looks good, why split it over several lines?

File: components/Admin/AdminForm.js [hunk 3]
```diff
@@ -221,7 +226,10 @@ class AdminForm extends Component {
             errors &&
             Object.prototype.hasOwnProperty.call(errors, 'images')
           ) {
-            return this.setState({ errors, updating: false });
```
Comment (nikrb) [inline components/Admin/AdminForm.js]: again, why split this to multiple lines?

**Review by nikrb — COMMENTED** [review]
Several instances where single line statement have been split over multiple lines? Why?

**Comment by g1st** [comment]
Sorry, that's my prettier on auto save. Let me try to fix it.

**Review by nikrb — APPROVED** [review]
LGTM

**Pull request merged**